### DISCUSSION TITLE
fnRandomSeed issue

### DISCUSSION THREAD

**Discussion by Ghost-dot98** [maecry/asphyxia-cs2#14] [General]
Hello maecry i was using ur project as a base for the cheat that im making and i have an issue when i turn bhop on. It crashes when i turn it on, i saw similar code in qo0's base but he was using different dll for it, idk is that because cs2 has different dll or something else. But anyway if u could help me fix this error it would be great.

Line of crash:
    Line 54: MATH::fnRandomSeed(pUserCmd->nRandomSeed);

when i remove this line it doesnt crash but also bhop stops working.

**Comment by siongadev** [comment]
This has been known since Value changed something and broke it. I'm sure @maecry will post a fix for it eventually.


Edit: That line is required for bhop to work (just comment it out and don't use bhop)

**Comment by maecry** [comment]
1. cusercmd updated and i havent pushed a fix as i dont have my pc w me rn
2. qo0 is used for csgo ofc the export from source1 is different from source2
3. Dicussion isnt a place for issues

**Comment by Ghost-dot98** [comment]
Okay, ty for response

ned, 3. dec 2023. 15:14 not maecry ***@***.***> је написао/ла:
